src/shared/lib/referenceGenomeUtils.ts

```diff
@@ -1,4 +1,10 @@
 export const referenceGenome = {
-    GRCH37: 'GRCh37',
-    GRCH38: 'GRCh38',
+    GRCH37: {
+        NCBI: 'GRCh37',
+        UCSC: 'hg19',
+    },
+    GRCH38: {
+        NCBI: 'GRCh38',
+        UCSC: 'hg38',
+    },
```

Comment:
Might be better to name the constant all uppercase, and keys lowercase:
https://github.com/airbnb/javascript#naming--uppercase

Reply:
@onursumer  for those keys, using lowercase would be a good name from naming side, but I think`NCBI` and `UCSC` is the abbreviation of those organization names, so uppercase might make sense from semantic side. So in this case should we make good naming a priority?
Uppercase for constant sounds good to me though.

Reply:
@onursumer 
```
export const REFERENCE_GENOME = {
    grch37: {
        NCBI: 'GRCh37',
        UCSC: 'hg19',
    },
    grch38: {
        NCBI: 'GRCh38',
        UCSC: 'hg38',
    },
};
```
how do you like this?